src/test/java/com/wikia/webdriver/common/core/api/GraphApi.java

```diff
@@ -1,6 +1,9 @@
 package com.wikia.webdriver.common.core.api;
 
+import com.jayway.jsonpath.DocumentContext;
+import com.jayway.jsonpath.JsonPath;
 import com.wikia.webdriver.common.core.XMLReader;
+import com.wikia.webdriver.common.core.helpers.FacebookUser;
 import com.wikia.webdriver.common.logging.PageObjectLogging;
 import com.google.gson.Gson;
 import com.google.gson.reflect.TypeToken;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.google.gson.reflect.TypeToken'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)